src/ui/win/win_ui.h

```diff
@@ -52,8 +54,27 @@ class UiWindowWin : public UiWindow
         PaintColor color;
     };
 
+    struct EllipseToDraw
+    {
+        EllipseToDraw( double xTopLeft_, double yTopLeft_, double xBottomRight_, double yBottomRight_, const PaintColor & color_ = PaintColor() )
+            : xTopLeft(xTopLeft_)
+            , yTopLeft(yTopLeft_)
+            , xBottomRight(xBottomRight_)
+            , yBottomRight(yBottomRight_)
+            , color(color_)
+        {
+        }
+
+        double xTopLeft;
```

Comment:
What if we name them as:
```cpp
double left;
double top;
double right;
double bottom;
```
? :)

Reply:
xTopLeft and yTopLeft refers to one point while xBottomRight and yBottomRight is another point.

Reply:
Then use `Point2d` structure for it.